docs/DeveloperGuide.adoc

```diff
@@ -299,16 +299,56 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a person |remove contacts that I no longer need
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |add a new group |create a group for scheduling meetings
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |user |add person to group |
+
+|`* * *` |user |delete a group |remove groups that I no longer need
+
+|`* * *` |user |find a group by name |locate details of groups without having to go through the entire list
+
+|`* * *` |user |import my current schedule |do not have to manually add my calendar events
+
+|`* * *` |user |import my friends' schedule easily |do not have to manually add their calendar events
+
+|`* * *` |user |view my schedule |see what's on my schedule
+
+|`* * *` |user |find a common free time between multiple schedules |schedule a meeting between multiple people quickly
+
+|`* * *` |user |schedule meetings with different intervals (multiple times a week, every week, biweekly) | arrange more regular meetings
+
+|`* * *` |user |import my current schedule |do not have to manually add my calendar events
+
+|`* * *` |user |add ad-hoc events |can de-conflict
```

Comment:
I think you can specify/ explain what **ad-hoc** is supposed to mean in this context.